src/pages/SonarFeed/SignalDetails.js

```diff
@@ -26,15 +30,16 @@ const SignalDetails = ({
   redirect,
   closeModal,
   id,
-  match = {}
+  match = {},
+  author = 'Santiment Team',
+  likesCount = 0
```

Comment:
I'm not sure, that default author name should be `Santiment Team` or another author. 
In figma author's name isn't showing, if it's your signal. But, if not - you need to show this. 

the same for delete button (show - if your). And maybe here we need any confirmation, discuss that with designer, pls